pandas/tests/io/xml/test_xml.py

```diff
@@ -782,6 +782,19 @@ def test_read_xml_passing_as_positional_deprecated(datapath):
         )
 
 
+@td.skip_if_no("lxml")
```

Comment:
Good test for `None` encoding. Can this new test be moved few lines earlier under previous *ENCODING* section (lines ~700-730). I wonder if we should also test for `etree` with `None` expecting success with `tm.assert_frame_equal`. etree tends to be more active in API so may change defaults with future versions.

Reply:
Added and moved